sensing/pointcloud_preprocessor/src/outlier_filter/dual_return_outlier_filter_nodelet.cpp

```diff
@@ -87,11 +87,6 @@ void DualReturnOutlierFilterComponent::filter(
   PointCloud2 & output)
 {
   boost::mutex::scoped_lock lock(mutex_);
-  if (input->data.empty()) {
-    output.header = input->header;
```

Comment:
Do you have any reasons why you delete header processing?

Reply:
Is this same reason as https://github.com/autowarefoundation/autoware.universe/pull/444/files/7a0875c0d729c024fc17da6fd82a1cb331f0b4cf#r814513436 ?

Reply:
same as https://github.com/autowarefoundation/autoware.universe/pull/444#discussion_r814513436